.dockerignore

```diff
@@ -1,13 +1,12 @@
 # config
-env.json
+**/env.*
 venv
 
 # data
-*.csv
+**/*.csv
 .data
 
 # etc
 .DS_Store
-*.pyc
-.vscode
-__pycache__
\ No newline at end of file
+**/*.pyc
```

Comment:
So the .dockerignore has a different syntax from .gitignore? That's kind of annoying, thanks for noticing this!

Reply:
Yeah, it's described near the bottom of this section: https://docs.docker.com/engine/reference/builder/#dockerignore-file